.github/workflows/ci-casper-rust-contract.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       matrix:
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-        node-version: [16.x]
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
NIT: Do we need to run CI against 3 versions? LTS - v20 would be enough.

Reply:
Problem is that it tries to validate previous 16.x....so we don't figure out how to remove 16.x and have only 20.x left

Reply:
hmm weird, then lets force merge this PR with v20. probably GitHub checks the workflow on the base branch too.

Reply:
@gyroflaw Can you please revert https://github.com/casper-ecosystem/cep18/pull/121/commits/960b2a80c928b1b959b19d4d3f60762a91c254d6 becasue ci/cd was passing with 16/20